fs/checksum_calculator.go

```diff
@@ -27,8 +33,23 @@ type calculatedFile struct {
 	err      error
 }
 
-// Sum returns a hex-encoded SHA256 checksum value of a file or directory given a path.
-func (c ChecksumCalculator) Sum(paths ...string) (string, error) {
+// SumAsHexString returns the sum of the underlying sha256 hash in hex format
+func (c ChecksumCalculator) SumAsHexString() string {
+	return hex.EncodeToString(c.sumValue)
+}
+
+// Write byte slice onto a shared Hash object that it can be summed later
+func (c ChecksumCalculator) Write(p []byte) (n int, err error) {
```

Comment:
Stylistically, I would prefer we didn't use named/bare returns. You'll see that we don't have them anywhere else in the `packit` codebase. Though this method is really small and easy to reason about, they tend to cause confusion about what is actually being returned in any non-trivial method implementation.